core/src/main/java/brooklyn/entity/basic/DynamicGroupImpl.java

```diff
@@ -163,13 +165,20 @@ public void onManagementBecomingMaster() {
         }
         setChangeListener = new MyEntitySetChangeListener();
         ((ManagementContextInternal) getManagementContext()).addEntitySetListener(setChangeListener);
-        try {
-            rescanEntities();
-        } catch (Exception e) {
-            log.warn("Error rescanning entities when rebinding; may be a group set against an unknown entity: "+e);
-            log.debug("Trace for rescan entities error", e);
-            Exceptions.propagateIfFatal(e);
-        }
+        Task<Object> rescan = Tasks.builder().name("rescan entities").body(
+            new Runnable() {
+                @Override
+                public void run() {
+                    try {
+                        rescanEntities();
+                    } catch (Exception e) {
+                        log.warn("Error rescanning entities when rebinding; may be a group set against an unknown entity: "+e);
```

Comment:
This is not just called on rebind - it's called when the entity is very created+managed as well.
Perhaps a better message would just be `"Error rescanning for members of "+DynamicGroupImpl.this+"; may be a ..."`